docs_src/getting-started/Ch-GettingStartedDockerUsers.md

```diff
@@ -11,8 +11,8 @@ If you wish to get the latest builds of EdgeX Docker images (prior to releases),
 
 ### Install Docker & Docker Compose
 
-To run Dockerized EdgeX, you need to install Docker. See
-<https://docs.docker.com/install/> to learn how to install
+To run Dockerized EdgeX, you need to install Docker first. See
+<https://docs.docker.com/engine/install/> to learn how to install
 Docker. If you are new to Docker, the same web site provides you
 educational information. The following short video is also very
 informative <https://www.youtube.com/watch?time_continue=3&v=VhabrYF1nms>
```

Comment:
modify the third sentance of line 21 to
See: <https://docs.docker.com/compose/> to learn more about Docker Compose and <https://docs.docker.com/compose/install/compose-plugin/> install it.

Reply:
udpated